docs/development.md

```diff
@@ -0,0 +1,60 @@
+Development environment
+======================
+
+The purpose of the development environment is only for running the different test and metric tools.
+
+To get started, first setup the development environment. The dev. env. is a virtual machine. Every task described is run
```

Comment:
~The dev. env.~

Avoid using shorthand abbreviations for improved readability. Or at least remove the `.` periods

Reply:
Fixed